src/components/Loader/Loader.tsx

```diff
@@ -1,8 +1,15 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import './Loader.scss';
+import { wait } from '../../api';
 
-export const Loader: React.FC = () => (
-  <div className="Loader" data-cy="loader">
-    <div className="Loader__content" />
-  </div>
-);
+export const Loader: React.FC = () => {
+  useEffect(() => {
+    wait(500);
+  });
```

Comment:
why did you add it?

Reply:
that was some hardly explainable move :)